js/damage.js

```diff
@@ -120,9 +120,13 @@ function getDamageResult(attacker, defender, move, field) {
     if (typeEffectiveness === 0) {
         return {"damage":[0], "description":buildDescription(description)};
     }
-    if (move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying" || (gen >= 6 && defender.weight >= 200))) {
+    if (move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying" || (gen >= 6 && defender.weight >= 200) || field.isGravity)) {
         return {"damage":[0], "description":buildDescription(description)};
     }
+    if (move.name === "Synchronoise" &&
```

Comment:
hmm I wish I didn't do this so late, I believe http://hastebin.com/qugizivayo.avrasm was the correct way to do this, can you confirm @Marty-D ?